Gem/Code/Source/HighInstanceExampleComponent.cpp

```diff
@@ -333,4 +336,22 @@ namespace AtomSampleViewer
         }
     }
 
+    void HighInstanceTestComponent::ResetNoClipController()
+    {
+        using namespace AZ;
+        using namespace AZ::Debug;
+        Camera::CameraRequestBus::Event(GetCameraEntityId(), &Camera::CameraRequestBus::Events::SetFarClipDistance, 2000.0f);
+        NoClipControllerRequestBus::Event(GetCameraEntityId(), &NoClipControllerRequestBus::Events::SetHeading, AZ::DegToRad(-44.722542));
+        NoClipControllerRequestBus::Event(GetCameraEntityId(), &NoClipControllerRequestBus::Events::SetPitch, AZ::DegToRad(24.987326));
```

Comment:
Why don't these need to be saved/restored like the far clip distance?

Reply:
I think the sample manager default resets these to 0